sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -43,6 +43,21 @@ public Breadcrumb(final @NotNull Date timestamp) {
     this.timestamp = timestamp;
   }
 
+  Breadcrumb(final @NotNull Breadcrumb breadcrumb) {
+    this.timestamp = breadcrumb.timestamp;
+    this.message = breadcrumb.message;
+    this.type = breadcrumb.type;
+    this.category = breadcrumb.category;
+    final Map<String, Object> dataClone = CollectionUtils.newConcurrentHashMap(breadcrumb.data);
+    if (dataClone != null) {
+      this.data = dataClone;
+    }
+    this.unknown = CollectionUtils.newConcurrentHashMap(breadcrumb.unknown);
+    final SentryLevel levelRef = breadcrumb.level;
+    this.level =
```

Comment:
I don't think this is actually needed for enums, is it @marandaneto?

Reply:
enum is a sort of singleton, I don't remember if I had an issue or just went ahead and "serialized" again, if you remove this and there are tests asserting that you don't mutate the copied value over, and tests are green, we can remove this